bff/src/Bff/Internal/DefaultAccessTokenRetriever.cs

```diff
@@ -1,13 +1,15 @@
 // Copyright (c) Duende Software. All rights reserved.
 // See LICENSE in the project root for license information.
 
-// ReSharper disable once CheckNamespace
-namespace Duende.Bff;
+
+using Duende.Bff.AccessTokenManagement;
+
+namespace Duende.Bff.Internal;
 
 /// <summary>
 /// Default implementation of IAccessTokenRetriever
 /// </summary>
-public class DefaultAccessTokenRetriever() : IAccessTokenRetriever
+internal class DefaultAccessTokenRetriever() : IAccessTokenRetriever
 {
     /// <inheritdoc />
     public virtual async Task<AccessTokenResult> GetAccessToken(AccessTokenRetrievalContext context)
```

Comment:
I don't think this needs to be virtual anymore

Reply:
true